### DISCUSSION TITLE
aspnet/6.0/bullseye-slim dependancies question

### DISCUSSION THREAD

**Discussion by deangoodmanson** [dotnet/dotnet-docker#3935] [Q&A]
regarding 
https://github.com/dotnet/dotnet-docker/blob/main/src/aspnet/6.0/bullseye-slim/amd64/Dockerfile#L16
# [ASP.NET](http://asp.net/) Core image
FROM $REPO:6.0.7-bullseye-slim-amd64

there isn't a https://github.com/dotnet/dotnet-docker/tree/main/src/runtime-deps/3.1/bullseye-slim for https://github.com/dotnet/dotnet-docker/tree/main/src/runtime-deps/6.0

does it inherit from 3.1, and is that intended and OK?

Related: https://github.com/dotnet/dotnet-docker/discussions/3695

I'm trying to figure out how to reference bullseye-full in my docker file for an aspnet service (and some related code), beyond:
FROM mcr.microsoft.com/dotnet/aspnet:6.0

**Comment by mthalman** [comment]
For precision, let's just refer to the AMD64 architecture. In that context, there is one `runtime-deps` container image for Debian 11 (Bullseye) that is shared by 3.1 and 6.0. That image is built by https://github.com/dotnet/dotnet-docker/blob/main/src/runtime-deps/3.1/bullseye-slim/amd64/Dockerfile. It's tagged as both a 3.1 and 6.0 image for usability purposes.

> I'm trying to figure out how to reference bullseye-full in my docker file for an aspnet service (and some related code), beyond:
> FROM mcr.microsoft.com/dotnet/aspnet:6.0

Meaning that you don't want `bullseye-slim` but rather just `bullseye`? Yes, in that case you'd simply modify the Dockerfile I just mentioned (https://github.com/dotnet/dotnet-docker/blob/main/src/runtime-deps/3.1/bullseye-slim/amd64/Dockerfile) and change `amd64/debian:bullseye-slim` to `amd64/debian:bullseye`.

I'm curious what need you have for the full version?

**Reply by deangoodmanson** [reply]
Thank you for a quick response.

> I'm curious what need you have for the full version?

We're installing an R runtime also. The services makes a Process call to it, and it is working fine for on OS X and Windows, but not in the Linux runtime.
